devices.js

```diff
@@ -11400,6 +11400,16 @@ const devices = [
             await configureReporting.rmsVoltage(endpoint);
         },
     },
+    // Tuya TYZS1L Led Strip Controller
+    {
+        zigbeeModel: ['TS0503A'],
+        model: 'TYZS1L',
```

Comment:
TYZS1L is the model name of the Zigbee module used not the device itself

Reply:
Right. Can't find the model so I guess we can just put it there likes "Led Controller TYZS1L".

Reply:
what's wrong with TS0503A which is the recognisable "model"